content/nextflow-on-aws-batch/30_batch/00_ami.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Create AMI"
+chapter: false
+weight: 01
+---
+
+AWS Batch uses ECS as an execution host and as such uses the official ECS-optimized image as a default.
+As the nextflow container needs to run the AWS-Cli we need to update the AMI so that we have everything we need.
+
+<!--
+## Paste Credentials
+
+In order to use packer, we need to paste the CLI credentials from the Event Engine Dashboard.
+Click [1] to copy the credentials in your clipboard and paste them into your Cloud9 terminal.
+
+![](/images/nextflow-on-aws-batch/batch/0_ee_dash.png)
+-->
+
+## Install Packer
+
+To update the image we use Hashicorps [packer]().
+
+```
+sudo yum install -y bsdtar jq
+curl -sLo - \
+      https://releases.hashicorp.com/packer/1.5.4/packer_1.5.4_linux_amd64.zip \
+      | sudo bsdtar xfz - -C /usr/bin/
+sudo chmod +x /usr/bin/packer
+```
+
+### Build image
```

Comment:
there are a few snippets on this page, make sure you tell the user what each one of them does